src/NHSD.BuyingCatalogue.Database/Tables/CatalogueItem.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE dbo.CatalogueItem
+(
+    CatalogueItemId varchar(14) NOT NULL,
+    [Name] varchar(255) NOT NULL,
+    CatalogueItemTypeId int NOT NULL,
+    SupplierId varchar(6) NOT NULL,
+    PublishedStatusId int CONSTRAINT DF_CatalogueItem_PublishedStatus DEFAULT 1 NOT NULL,
+    Created datetime2(7) CONSTRAINT DF_CatalogueItem_Created DEFAULT GETUTCDATE() NOT NULL,
+    CONSTRAINT PK_CatalogueItem PRIMARY KEY (CatalogueItemId),
```

Comment:
Is the default clustered or nonclustered for a primary key constraint? Does it need to be defined here?

Reply:
The default is clustered, so in this case the PK is clustered. I would normally only include the keyword for non-clustered PKs but if including the keyword is clearer for others then we can add it.